cube/bmp_8.cpp

```diff
@@ -77,14 +73,14 @@ unsigned char **Bmp8::CreateBmpBuffer(unsigned char init_val) {
 
   buff = (unsigned char **) new unsigned char *[hgt_ * sizeof(*buff)];
   if (!buff) {
```

Comment:
cube will be removed from source code... I would like to keep it for 3.05 at least. 
So only bugfixes for cube make sense for me..